src/main/java/com/training/assignments/miniproject/EmployeeManagement.java

```diff
@@ -0,0 +1,18 @@
+package com.training.assignments.miniproject;
+
+import lombok.Data;
+
+@Data
+public class EmployeeManagement {
```

Comment:
Why are you complicating this by giving this name?
This should be as simple as Employee